pkgs/applications/misc/onboard/default.nix

```diff
@@ -91,6 +91,7 @@ in python3.pkgs.buildPythonApplication rec {
     wrapGAppsHook
     xorg.libXtst
     xorg.libxkbfile
+    gobject-introspection # Temporary fix, see https://github.com/NixOS/nixpkgs/issues/56943
```

Comment:
```suggestion
```

And use
```
strictDeps = false;
```
with the comment.

Reply:
@worldofpeace is it better to remove `strictDeps` vs listing twice? If so, why?

Reply:
The issue is actually triggered preventing the setup hook from being ran because of strictDeps being used https://github.com/NixOS/nixpkgs/issues/56943#issuecomment-470059976.

Reply:
So this will be the best work around until the eventual issue is fixed.

Reply:
Correct, as python packages enabled `strictDeps` this was the regression that followed for gobject-introspection using applications.